nimbus-test/src/main/java/com/antheminc/oss/nimbus/test/scenarios/chart/view/VRSampleChart.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * 
+ */
+package com.antheminc.oss.nimbus.test.scenarios.chart.view;
+
+import java.util.List;
+
+import com.antheminc.oss.nimbus.domain.defn.Domain;
+import com.antheminc.oss.nimbus.domain.defn.Domain.ListenerType;
+import com.antheminc.oss.nimbus.domain.defn.Execution.Config;
+import com.antheminc.oss.nimbus.domain.defn.Repo;
+import com.antheminc.oss.nimbus.domain.defn.Repo.Cache;
+import com.antheminc.oss.nimbus.domain.defn.Repo.Database;
+import com.antheminc.oss.nimbus.entity.aggregate.chart.DataGroup;
+
+import lombok.Getter;
+import lombok.Setter;
+
+/**
+ * @author Rakesh Patel
+ *
+ */
+@Domain(value="samplechartview", includeListeners= {ListenerType.websocket})
+@Repo(value= Database.rep_none, cache=Cache.rep_device)
+@Getter @Setter
+public class VRSampleChart {
+	
+	/*
+	 * Below configuration is only for test case purpose. When @Chart is used for view configuration in live applications, 
+	 * please look at PetClinic implementaion for configuration options
+	 */
+	private List<DataGroup> actionPerDomainRoot;
```

Comment:
Does the DataGroup have to be a List<> always? If I dont have multiple datagroups to plot, can I just use -
Private DataGroup actionPlot;

Reply:
The server side contract can be flexible in terms of param type being List<DataGroup> or DataGroup depending on the uasge. 
**Example**: if you use aggregate query to get the chart data then you need to use List<DataGroup> since it would always return list whereas if you are writing a simple query against one collection (that collection can be a view also) then just DataGroup can be used with fetch=1 clause

@sandeepmantha  could you please confirm if the client side (angular) would we be able to handle DataGroup or List<DataGroup> ?

Reply:
Updated the SampleChartDataSetTest to add a test method for this

Reply:
@dinakarmeda @rakeshnpatel - the leafState does not get generated for a complex object. In this case its DataGroup. So if the user wants to generate just 1 graph, he needs to project the data as a collection and in this case it would be 1 collection element.

Reply:
The UI would understand collections(List<DataGroup>) for a chart and render 1 or many graphs.